.github/workflows/node.js.yml

```diff
@@ -0,0 +1,70 @@
+name: fail_fast_CI
+on:
+  push:
+    branches:
+      - dev
+      - feature    
+      - ktchethan-patch-1
+  pull_request:
+  release:
+    types:
+      - staging
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  checkout_and_install:
+    strategy:
+      matrix:
+         node: ["15.14.0"]        
+        #node: ["12.22.1", "14.16.1", "15.14.0", "16.1.0"]        
+    runs-on: ubuntu-latest    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}          
+      - name: Cache node modules
```

Comment:
This part is duplicated within the checkmycode.yml
How can we resolve this duplication?

Reply:
Their are two different files checkmycode.yml and node.js.yml, here node.js.yml used for local testing and test the  design implementation. checkmycode.yml was designed and instructured to be the final expected workflow but it was not tested.